UI/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Root folder and module folder like UI, Chart, Engine... are not project because no jar will be generated from this folder. We
What is the reason to add .project file here?

Reply:
The .project files are generated by Eclipse when importing the maven projects.